site/docs/pages/getting-started.mdx

```diff
@@ -42,54 +42,53 @@ npm i @coinbase/onchainkit@latest @wagmi/core@latest
 NEXT_PUBLIC_ONCHAINKIT_API_KEY=YOUR_PUBLIC_API_KEY
 ```
 
-### Set Up Onchain Providers
-
-Set up `OnchainProvider` and `WagmiProvider`. 
+## Add OnchainKitProvider
 
-Inside the `WagmiProvider`, wrap your app in a TanStack Query React Context Provider, e.g. `QueryClientProvider`, and pass a new `QueryClient` instance to the `client` property.
+In your `providers.tsx` file, add `OnchainKitProvider` as a child of `WagmiProvider` and `QueryClientProvider`.
 
+Additionally, add Base as a supported chain in the Wagmi configuration file `wagmi.ts`.
 
 :::code-group
 
-```tsx twoslash [OnchainProviders.tsx]
+```tsx twoslash [providers.tsx]
 // @noErrors: 2307 2580 - cannot find 'process', cannot find './wagmi'
-'use client';
-import { ReactNode } from 'react';
 import { OnchainKitProvider } from '@coinbase/onchainkit';
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
-import { base } from 'viem/chains';
-import { WagmiProvider } from 'wagmi';
-import { wagmiConfig } from './wagmi';
+import { base } from 'wagmi/chains'; // [!code ++]
```

Comment:
Should this be imported from viem to match the OnchainKitProviderReact type?

Reply:
I'm importing in my demo app from `wagmi/chains` with no issue

Reply:
I think we should standardize on `wagmi` considering we use wagmi in the rest of the files